### ISSUE TITLE
A shadow of old bug?

### ISSUE THREAD

**Issue by aerohub** [LouisBarranqueiro/hexo-theme-tranquilpeak#419]
### Configuration

 - **Tranquilpeak version** : 
 Not modified in these parts.

Hello Louis! There was an old issue https://github.com/LouisBarranqueiro/hexo-theme-tranquilpeak/issues/46 It was fixed temporarily for page pagination layout. Works fine on second page of the whole blog when we need to open the main (the first) page of it. It seems the main blog url is generated for the "NEWER" link, isn't it?

But this fix doesn't work properly when we need to paginate for example tags pages the same way. Lets say we have a lot of posts with some tag. When pagination is turned on in configuration file we get pages for the tag:

```
http://domain.org/tags/sometag/
http://domain.org/tags/sometag/page/2/
http://domain.org/tags/sometag/page/3/
...
```
The problem is in that the same pagination layout is in use. And when we click the NEWER button on the `http://domain.org/tags/sometag/page/2/` we'll go to the main page of the blog not to `http://domain.org/tags/sometag/`

I get this behaviour either on local or on host. Suppose that is the same for pages in `/categories/`

**Comment by LouisBarranqueiro** [comment]
Hey @aerohub, i'm on it!

**Comment by LouisBarranqueiro** [comment]
Fixed in v1.11.0
